modules/ROOT/pages/configuring-user-analytics.adoc

```diff
@@ -0,0 +1,67 @@
+= Configuring User Analytics 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use Google Analytics to gain valuable insights into how consumers are using your APIs and portal, identify areas for improvement, make data-driven decisions to optimize the user experience, and drive business success.
+
+Google Analytics tracks the following data:
+
+API usage metrics: 
+
+* API instance access requests
+* API specification downloads
+* Most viewed API
+* Total number of calls made to an API
+
+User engagement metrics:
+
+* Active users
+* Login events
+
+API Console usage:
+
+* API requests
++ 
+Measure the overall API usage and identity patterns in API consumption. Captures each time an API request is made through the API Console and the total number of API requests made.
+* API usage by endpoint
++
+Identities popular endpoints and monitors usage trends to optimize API design and resource allocation. Captures each time an API request is made to a specific endpoint through the API console and the number of requests made to each API endpoint.
+
+== Configure Google Analytics
+To configure Google Analytics to send events from the portal:
+
+. Set up a Google Analytics property using the instructions in the https://support.google.com/analytics/answer/1008080?hl=en&visit_id=638531259961972871-3450454930&rd=1#zippy=%2Cin-this-article[Set up the Analytics Google tag] topic from the Google documentation.
+. Navigate to the Builder.
+. Click *Settings* > *Security & Privacy*.
```

Comment:
This is a good way to combine steps.

Fewer steps = less complex (or the appearance of)